source/_authors/crell.twig

```diff
@@ -0,0 +1,9 @@
+---
+title: Larry Garfield
+identifier: crell
+twitter: crell
+avatar: # TODO - REPLACE
+bio: Director of Developer Experience at Platform.sh. Collaboration advocate. Clean code advocate. Clean government advocate. Geek. #PSR8-compatible.
```

Comment:
```suggestion
bio: Larry Garfield is an aspiring blacksmith who moonlights as Director of Developer Experience for Platform.sh.  When not trying to hand-forge his own medieval armory from scratch he tries to teach developers and development managers the skills of yesteryear that the industry has forgotten. #PSR8 compatible.
```

That's the version I've been using for comical bios.  If it's too long then the Twitter bio is fine. :-)